client/src/pages/Home.js

```diff
@@ -26,16 +26,25 @@ export function Home() {
 	return (
 		<main role="main">
 			<div>
-				<img
+				{/* <img
```

Comment:
We usually don't commit commented code because future developers don't know if it's safe to delete it, and it also makes the code messier. Can we delete this commented code?

Reply:
We can deleted is not usefull